Simplenote/AccountRemote.swift

```diff
@@ -51,4 +51,94 @@ class AccountRemote: Remote {
 
         return request
     }
+
+    // MARK: - Passkeys
+    //
+    private func passkeyCredentialCreationRequest(withEmail email: String, password: String) -> URLRequest {
+        let params = [
+            "email": email.lowercased(),
+            "password": password,
+            "webauthn": "true"
+        ] as [String: Any]
+
+        let boundary = "Boundary-\(UUID().uuidString)"
+        var request = URLRequest(url: SimplenoteConstants.passkeyCredentialCreationURL)
+        request.addValue("multipart/form-data; boundary=\(boundary)", forHTTPHeaderField: "Content-Type")
+
+        request.httpMethod = RemoteConstants.Method.POST
+        request.httpBody = body(with: boundary, parameters: params)
+
+        return request
+    }
+
+    private func body(with boundary: String, parameters: [String: Any]) -> Data {
+        var body = Data()
+
+        for param in parameters {
+            let paramName = param.key
+            body += Data("--\(boundary)\r\n".utf8)
+            body += Data("Content-Disposition:form-data; name=\"\(paramName)\"".utf8)
+            let paramValue = param.value as! String
+            body += Data("\r\n\r\n\(paramValue)\r\n".utf8)
+        }
+
+        body += Data("--\(boundary)--\r\n".utf8)
+
+        return body
```

Comment:
Rather than converting every line into `Data`, adding them all up and returning, we can build the string and convert just once, in the last statement